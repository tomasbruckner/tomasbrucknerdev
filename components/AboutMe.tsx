import React from 'react';
import { Typography } from '@material-ui/core';
import { useTranslation } from 'react-i18next';

const AboutMe = () => {
  const { t } = useTranslation();

  return (
    <div>
      <Typography id="about" variant="h4" gutterBottom style={{ marginTop: '1em' }}>
        {t('navigation.about')}
      </Typography>
      <Typography>{t('about.text')}</Typography>
    </div>
  );
};

export default AboutMe;
