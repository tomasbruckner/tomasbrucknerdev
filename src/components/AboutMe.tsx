import React, { FC } from 'react';
import { Typography } from '@material-ui/core';
import { t } from '../utils/i18n';

const AboutMe: FC = () => {
  return (
    <div>
      <Typography id="about" variant="h4" gutterBottom>
        {t('navigation.about')}
      </Typography>
      <Typography>{t('about.text')}</Typography>
    </div>
  );
};

export default AboutMe;
