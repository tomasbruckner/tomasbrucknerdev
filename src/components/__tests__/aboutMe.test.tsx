jest.mock('react-i18next', () => ({
  useTranslation: () => ({
    t: (text: string) => text,
  }),
}));

import React from 'react';
import AboutMe from 'components/AboutMe';
import renderer from 'react-test-renderer';

it('AboutMe renders correctly', () => {
  const tree = renderer.create(<AboutMe />).toJSON();
  expect(tree).toMatchSnapshot();
});
