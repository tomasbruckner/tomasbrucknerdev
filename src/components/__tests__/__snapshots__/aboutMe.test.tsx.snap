// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe renders correctly 1`] = `
<div>
  <h4
    className="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom"
    id="about"
  >
    navigation.about
  </h4>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    about.text
  </p>
</div>
`;
